scripts/docker/fedora.24/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM fedora:24
 
-RUN dnf install -y bash git cmake wget which python clang-3.8.1-1.fc24.x86_64 llvm-devel-3.8.1-2.fc24.x86_64 make libicu-devel lldb-devel.x86_64 \
+RUN dnf install -y bash git cmake wget which python clang-3.8.0-1.fc24.x86_64 llvm-devel-3.8.0-1.fc24.x86_64 make libicu-devel lldb-devel.x86_64 \
```

Comment:
I looked around but didnt come across any release notes for the differences. Looking at https://pkgs.org/download/clang for Fedora 24, 3.8.0-1 is the only option, so it seems like that is the only thing we can work with.

> Docker image can be rebuilt in the future if needed and produce an image with the same components.

Given the above, do you want to fix Fedora 23 dockerfile as well?